packages/design-system-docs/src/pages/components/UsaBanner/usabanner.example.html

```diff
@@ -0,0 +1,172 @@
+        class="ds-c-usa-banner__header-flag"
+        aria-labelledby="banner-flag-title-1 banner-flag-description-1"
+        focusable="false"
+      >
+        <title id="banner-flag-title-1">Flag</title>
+        <desc id="banner-flag-description-1">U.S. flag</desc>
+        <use href="{{root}}/images/usa-banner-flag.svg#svg"></use>
+      </svg>
+    </p>
+    <p class="ds-c-usa-banner__header-text">
+      <span>An official website of the United States government</span>
+      <span class="ds-c-usa-banner__header-action" aria-hidden="true">Here’s how you know</span>
+      <button class="ds-c-usa-banner__button" aria-expanded="true" aria-controls="gov-banner">
+        <span class="ds-c-usa-banner__button-text">Here’s how you know</span>
+      </button>
+    </p>
+  </header>
+  <div class="ds-c-usa-banner__content" id="gov-banner">
+    <div
+      class="ds-u-display--flex ds-u-flex-direction--column ds-u-sm-flex-direction--row ds-u-flex-wrap--nowrap"
+    >
+      <div class="ds-c-usa-banner__guidance">
+        <svg
+          class="ds-c-usa-banner__icon"
+          xmlns="http://www.w3.org/2000/svg"
+          viewBox="0 0 54 54"
+          aria-hidden="true"
+        >
+          <use href="{{root}}/images/usa-banner-dot-gov.svg#svg"></use>
+        </svg>
+        <p class="ds-c-usa-banner__media-body">
+          <strong>Official websites use .gov</strong>
+          <br />
+          A
+          <strong> .gov </strong>
+          website belongs to an official government organization in the United States.
+        </p>
+      </div>
+      <div class="ds-c-usa-banner__guidance">
+        <svg
+          class="ds-c-usa-banner__icon"
+          xmlns="http://www.w3.org/2000/svg"
+          viewBox="0 0 54 54"
+          aria-hidden="true"
+        >
+          <use href="{{root}}/images/usa-banner-https.svg#svg"></use>
+        </svg>
+        <p class="ds-c-usa-banner__media-body">
+          <strong>Secure .gov websites use HTTPS</strong>
+          <br />
+          A
+          <strong>
+            lock
+          </strong>
+          (
+          <svg
+            role="img"
+            xmlns="http://www.w3.org/2000/svg"
+            viewBox="0 0 52 64"
+            class="ds-c-usa-banner__lock-image"
+            aria-labelledby="banner-lock-title-1 banner-lock-description-1"
+            focusable="false"
+          >
+            <title id="banner-lock-title-1">Lock</title>
+            <desc id="banner-lock-description-1">A locked padlock</desc>
+            <use href="{{root}}/images/usa-banner-lock.svg#svg"></use>
+          </svg>
+          ) or
+          <strong> https:// </strong>
+          means you’ve safely connected to the .gov website. Share sensitive information only on
+          official, secure websites.
+        </p>
+      </div>
+    </div>
+  </div>
+</section>
+<h6 class="preview__label">Spanish banner</h6>
+<section class="ds-c-usa-banner" aria-label="Official government website">
```

Comment:
It seems like the `aria-label` here should also be in Spanish. Also, we should add a `lang="es"` to the `section` element so assistive tech knows what language to use.

Reply:
Thanks @davidakennedy - I have updated with lang attribute and change aria-label to Spanish (which I manually used google translation to get the translation "Sitio web oficial del gobierno"). Does it look right?

I have notice that the 'Flag' and 'Lock' image for Spanish Banner are also read in English. What's your thought on this? Should we add another a Spanish version for these SVG files? Or, we can keep it as is for now and update when USWDS does?

Reply:
cc: @bernardwang @line47

Reply:
Yeah, it's odd that the USWDS banner doesnt do this. We should file a ticket or PR for USWDS on this. In the mean time I think to handle it ourselves on our side

Reply:
`Sitio web oficial del gobierno`
Flag: `Bandera`
Lock: `Candado`

Personally, I think this could be handled in a followup so we can confirm these translations, but since this is a beta component I think its fine to just quickly add it here as well

Reply:
I wonder about the images in the banner and the need for aria-labels because they all seem to be decorative and could easily be skipped and no context would be lost - thoughts @davidakennedy

Reply:
@line47 Which ones do you mean?